dotnet/src/SemanticKernel.Abstractions/Functions/EventDelegateWrapper{TEventArgs}.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.SemanticKernel.Events;
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the main namespace
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Class for storing event handler and event args for function events.
+/// </summary>
+/// <typeparam name="TEventArgs"></typeparam>
+public sealed class EventHandlerWrapper<TEventArgs> where TEventArgs : SKEventArgs
```

Comment:
Class name doesn't match file name.
Class name: `EventHandlerWrapper<TEventArgs>`.
File name: `EventDelegateWrapper{TEventArgs}.cs`

Reply:
We can't have file names with `<>`, Renaming it to just the `EventHandlerWrapperOfT` as the one of the approach used in the [dotnet runtime repo](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs).